Comparer/CompareExpressions.cs

```diff
@@ -1,10 +1,19 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
+﻿using System.Collections.Generic;
 
 namespace Project2_Group_17.Comparer
 {
-    public class CompareExpressions
+    public class CompareExpressions : IComparer<string>
     {
+        /// <summary>
+        /// Compares the two results to ensure they are the same
+        /// </summary>
+        /// <param name="x">A result</param>
+        /// <param name="y">A result</param>
+        /// <returns>1 if they are the same, 0 otherwise</returns>
+        public int Compare(string x, string y)
+        {
+            if (x == y) return 1;
+            else return 0;
+        }
     }
```

Comment:
typically Compare is suppose to return 1, 0 and -1 for a greater/equal/less than b.

I think this does serve the only purpose we need it to for the project, but probably best to do it as intended just in case he might doc marks for this

Reply:
So basically the alternative to this is using the string CompareTo method. Which if we do that, it kind of defeats the purpose of implementing the IComparer interface (since strings cant be compared with < or >). So I can change the numeric values to be closer to other comparing methods but otherwise its going to end up being very similar to what it currently is.

Reply:
hmm a good point, this is a weird thing for him to ask us to do when it's a basic available function. I think keeping it like this makes fine sense then. Unless he wants us to be running evaluate on each and comparing the number not the string?

Reply:
Its to check a box tbh but I updated it to compare doubles

Reply:
yeah man idk what he wants, may try to remember to ask tomorrow but this may definitely just be a box checker